internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -347,25 +366,27 @@ func ParseSessionLogRecord(c SessionLogRecord) (spec sdktypes.SessionLogRecord,
 }
 
 type SessionCallSpec struct {
-	SessionID sdktypes.UUID `gorm:"primaryKey:SessionID;type:uuid;not null"`
-	Seq       uint32        `gorm:"primaryKey"`
-	Data      datatypes.JSON
+	SessionID      sdktypes.UUID `gorm:"primaryKey:SessionID;type:uuid;not null"`
+	Seq            uint32        `gorm:"primaryKey"`
+	Data           datatypes.JSON
+	CompressedData []byte
 
 	// enforce foreign keys
 	Session *Session
 }
 
 func ParseSessionCallSpec(c SessionCallSpec) (spec sdktypes.SessionCallSpec, err error) {
-	err = json.Unmarshal(c.Data, &spec)
+	err = unmarshalData(&spec, c.Data, c.CompressedData)
 	return
 }
 
 type SessionCallAttempt struct {
-	SessionID sdktypes.UUID `gorm:"uniqueIndex:idx_session_id_seq_attempt,priority:1;type:uuid;not null"`
-	Seq       uint32        `gorm:"uniqueIndex:idx_session_id_seq_attempt,priority:2"`
-	Attempt   uint32        `gorm:"uniqueIndex:idx_session_id_seq_attempt,priority:3"`
-	Start     datatypes.JSON
-	Complete  datatypes.JSON
+	SessionID          sdktypes.UUID `gorm:"uniqueIndex:idx_session_id_seq_attempt,priority:1;type:uuid;not null"`
+	Seq                uint32        `gorm:"uniqueIndex:idx_session_id_seq_attempt,priority:2"`
+	Attempt            uint32        `gorm:"uniqueIndex:idx_session_id_seq_attempt,priority:3"`
+	Start              datatypes.JSON
+	Complete           datatypes.JSON
+	CompressedComplete []byte
```

Comment:
Use a bool for weather `Complete` is compressed or not. Data type of `Complete` should be `[]byte`